bindata/network/ovn-kubernetes/ovnkube-node.yaml

```
@@ -274,6 +299,9 @@ spec:
             --loglevel "${OVN_KUBE_LOG_LEVEL}" \
             --inactivity-probe="${OVN_CONTROLLER_INACTIVITY_PROBE}" \
             ${gateway_mode_flags} \
+            {{ if eq .OVN_DPU_MODE "dpu-host" }}
+            --ovnkube-node-mode dpu-host \
+            {{ end }}
```

```suggestion
            {{- end }}
```
White line was being added and cause script to break.